actors/builtin/miner/miner_state.go

```diff
@@ -692,6 +693,50 @@ func (st *State) CheckSectorHealth(store adt.Store, dlIdx, pIdx uint64, sector a
 	return nil
 }
 
+// Returns an error if the target sector cannot be found and/or is faulty/terminated/unproven.
+func (st *State) CheckSectorHealthExcludeUnproven(store adt.Store, dlIdx, pIdx uint64, sector abi.SectorNumber) error {
```

Comment:
what's the difference between this and `CheckSectorProven` ?

Reply:
it also checks that the sector isn't faulty